wasmer/import.go

```diff
@@ -178,6 +178,13 @@ type InstanceContext struct {
 	memory  Memory
 }
 
+func NewInstanceContext(ctx *cWasmerInstanceContextT, mem Memory) *InstanceContext {
```

Comment:
So only used in tests, and always called with first parameter `nil`. Perhaps make a file `utils_test.go` and it will be shared among the tests? Or no.

Reply:
The fields of `InstanceContext` are currently private, so it can't be instantiated from outside the `wasmer` package.